### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by omkelderman**
some small refactor things, update dependencies, and a typed config system

### CODE REVIEW

File: modules/database.js [hunk 1]
```diff
@@ -25,20 +25,28 @@ function initPlayersTable(db) {
     console.log("Created players table");
 
     console.log("Importing players from players.tsv");
+
+    const promisisArray = [];
```
Comment (IceDynamix) [inline modules/database.js]:
```suggestion
    const promisesArray = [];
```
don't forget to refactor

**Review by IceDynamix — CHANGES_REQUESTED** [review]

File: modules/database.js (hunk 1 above)
Comment (omkelderman) [reply modules/database.js]: oh hahahaha nice typo 😂 

thats gonna be fun for me, I branched this of a commit thats way down my own commit history so uh, guess I'll have to merge this on top of my other stuff to stay up to date 🤣  oh well shouldnt be too much of a problem xd

**Comment by IceDynamix** [comment]
You also have to adjust the usages too, not just the declaration lol

**Comment by omkelderman** [comment]
bruh, I just clicked the button accept changes assuming that was all that was needed lol, I dont have the code in front of me, guess I'll do that after work xd

**Comment by omkelderman** [comment]
force pushed a thing, I think its ok now? 👀

**Pull request merged**